chapter02/dtj/ex5

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/perl
+
+use warnings;
+
+print "Enter the string: ";
+$string = <STDIN>;
+print "Enter the number of repeats: ";
+chomp($num = <STDIN>);
+print $string x $num;
```

Comment:
Hi @dpratyusha , As far as I remember it does print on separate lines, the reason being that there's no chomp on line 6, so the string that gets entered retains it's "\n". 

(It's not particularly clear, so the sensible thing to do would have probably been to comment it....)